### ISSUE TITLE
Migration to version 9.0

### ISSUE THREAD

**Issue by pedrobaeza** [OCA/connector-woocommerce#10]
# Todo

https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-9.0
# Modules to migrate
- [ ] connector_woocommerce

**Comment by yvaucher** [comment]
Is this wrongly checked as migrated to 9.0 version ?

The module is uninstallable https://github.com/OCA/connector-woocommerce/blob/9.0/connector_woocommerce/__openerp__.py#L31 and I don't see any PR

**Comment by pedrobaeza** [comment]
Yes, indeed, the migration is in progress by @parthivgls, but no PR yet.
